apps/accounts/migrations/0002_add_jwt_access_token_model.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.2.13 on 2021-01-01 06:23
```

Comment:
It seems like the `pre-commit` hook for black and flake8 isn't working. Can you please install the hook and use it.

Reply:
@RishabhJain2018 no files were updated after running `pre-commit`

Reply:
Just confirming, these are the steps that you followed --
1. `pre-commit install`
2. `pre-commit run -a`